maintenance_project/views/maintenance_equipment_views.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="hr_equipment_view_search" model="ir.ui.view">
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_search" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='owner_user_id']" position="after">
+                <field name="project_id"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="hr_equipment_view_tree" model="ir.ui.view">
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='category_id']" position="after">
+                <field name="project_id"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="hr_equipment_view_kanban" model="ir.ui.view">
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_kanban" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='activity_state']" position="after">
+                <field name="project_id" />
+            </xpath>
+            <xpath expr="//div[@class='o_kanban_record_body']" position="inside">
+                <div t-if="record.project_id.raw_value">
+                    <small>Project: <t t-raw="record.project_id.value"/></small>
+                </div>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="hr_equipment_view_form" model="ir.ui.view">
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='location']" position="after">
+                <field name="create_project_from_equipment" attrs="{'invisible': [('id', '!=', False)]}"/>
+                <field name="project_id" attrs="{'invisible': [('create_project_from_equipment', '=', True), ('id', '=', False)]}"/>
+            </xpath>
+            <xpath expr="//div[field[@name='maintenance_duration']]" position="after">
```

Comment:
There's a conflict with module `maintenance_plan` that causes field `preventive_default_task_id` not be visible. Could you check?

I think it would fixed if you change the view for `preventive_default_task_id`, something like this:

```
            <xpath expr="//group[@name='maintenance']" position="after">
                <group name="project_task">
		    <field name="preventive_default_task_id"
                           domain="[('project_id', '=', project_id)]"
                           context="{'default_project_id': project_id}"/>
                </group>
            </xpath>
```

Reply:
Done... but It's time to think about a `maintenance_project_plan` module because of this feature

Reply:
Do you think the field `preventive_default_task_id` would be used by `maintenance_project_plan` only? In that case I think would be better to remove the field from here and put it directly in `maintenance_project_plan`.

Reply:
`preventive_default_task_id` is already in use here (a maitenance request generated by the cron will take it), but later, when you install `maintenance_plan`, this field should be "moved" with the other preventive fields to the plans. So, I realized a `maintenance_project_plan ` module is also needed.